rust/headless-client/src/dns_control.rs

```diff
@@ -8,8 +11,61 @@ mod windows;
 #[cfg(target_os = "windows")]
 use windows as platform;
 
-pub(crate) use platform::{deactivate, system_resolvers, DnsController};
+pub(crate) use platform::system_resolvers;
 
 // TODO: Move DNS and network change listening to the IPC service, so this won't
 // need to be public.
 pub use platform::system_resolvers_for_gui;
+
+pub(crate) struct DnsController {
+    /// True if DNS control is definitely active or might be active.
+    ///
+    /// In case the IPC service has crashed or something, we always assume that DNS control
+    /// is active when we start. Deactivating Firezone's DNS control is safe, but it takes
+    /// a lot of time on Windows, so we'd like to avoid redundant de-activations.
+    control_may_be_active: bool,
```

Comment:
How about removing the `may` from the name and explaining the default with the comment instead?

Reply:
I changed it to just `in_control`